internal/doculint/doculint.go

```diff
@@ -297,11 +297,11 @@ func validateGenDeclConstants(reporter nolint.Reporter, expr *ast.GenDecl) {
 // that if it is a type declaration block that the block itself has a comment, and each
 // type declaration within it also has a comment. If it is a standalone type declaration
 // it ensures that it has a comment associated with it.
-func validateGenDeclTypes(reporter nolint.Reporter, expr *ast.GenDecl) {
+func validateGenDeclTypes(report reporter.Reporter, expr *ast.GenDecl) {
```

Comment:
"I'm a reporter."

"What should I call you?"

"Report."

"You're a _report_?"

"No! I'm a reporter. I just like to be called 'report'."

Reply:
Actually @george-e-shaw-iv this should be `r` because the type is self describing

Reply:
good callout, changed